deltachat-ios/Controller/AccountSetupController.swift

```diff
@@ -493,112 +530,143 @@ class AccountSetupController: UITableViewController {
         }
-
+        
         print("oAuth-Flag when loggin in: \(DcConfig.getAuthFlags())")
         dc_configure(mailboxPointer)
         showProgressHud(title: String.localized("login_header"))
     }
-
+    
     @objc func closeButtonPressed() {
         dismiss(animated: true, completion: nil)
     }
-
+    
     // returns true if needed
     private func showOAuthAlertIfNeeded(emailAddress: String, handleCancel: (() -> Void)?) -> Bool {
         return false
-
+        
         // disable oauth2 for now as not yet supported by deltachat-rust.
         /*
          if skipOauth {
-         	// user has previously denied oAuth2-setup
-         	return false
+         // user has previously denied oAuth2-setup
+         return false
          }
-
+         
          guard let oAuth2UrlPointer = dc_get_oauth2_url(mailboxPointer, emailAddress, "chat.delta:/auth") else {
-         	//MRConfig.setAuthFlags(flags: Int(DC_LP_AUTH_NORMAL)) -- do not reset, there may be different values
-         	return false
+         //MRConfig.setAuthFlags(flags: Int(DC_LP_AUTH_NORMAL)) -- do not reset, there may be different values
+         return false
          }
-
+         
          let oAuth2Url = String(cString: oAuth2UrlPointer)
-
+         
          if let url = URL(string: oAuth2Url) {
-         	let title = "Continue with simplified setup"
-         	// swiftlint:disable all
-         	let message = "The entered e-mail address supports a simplified setup (oAuth2).\n\nIn the next step, please allow Delta Chat to act as your Chat with E-Mail app.\n\nThere are no Delta Chat servers, your data stays on your device."
-
-         	let oAuthAlertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
-         	let confirm = UIAlertAction(title: "Confirm", style: .default, handler: {
-         		[unowned self] _ in
-         		let nc = NotificationCenter.default
-         		self.oauth2Observer = nc.addObserver(self, selector: #selector(self.oauthLoginApproved), name: NSNotification.Name("oauthLoginApproved"), object: nil)
-         		self.launchOAuthBrowserWindow(url: url)
-         	})
-         	let cancel = UIAlertAction(title: "Cancel", style: .cancel, handler: {
-         		_ in
-         		MRConfig.setAuthFlags(flags: Int(DC_LP_AUTH_NORMAL))
-         		self.skipOauth = true
-         		handleCancel?()
-
-         	})
-         	oAuthAlertController.addAction(confirm)
-         	oAuthAlertController.addAction(cancel)
-
-         	present(oAuthAlertController, animated: true, completion: nil)
-         	return true
+         let title = "Continue with simplified setup"
+         // swiftlint:disable all
+         let message = "The entered e-mail address supports a simplified setup (oAuth2).\n\nIn the next step, please allow Delta Chat to act as your Chat with E-Mail app.\n\nThere are no Delta Chat servers, your data stays on your device."
```

Comment:
also here  @nayooti your editor is doing strange things and reformats unaffected code. i am currently fixing that, just saying.

wrt. the empty lines, in xcode, you have to enable "Preferences / Text Editting / Editing / Including whitespace only lines"